frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -63,33 +31,82 @@ export default function ProgramOverview() {
     const [selectedSections, setSelectedSections] = useState<number[]>([]);
     const [searchTerm, setSearchTerm] = useState<string>('');
     const [sortQuery, setSortQuery] = useState<string>('name_last asc');
-    const [includeActive, setIncludeActive] = useState(true);
+    const [includeArchived, setIncludeArchived] = useState(false);
+    const archiveSectionsRef = useRef<HTMLDialogElement>(null);
+    const [unableToArchiveSections, setUnableToArchiveSections] = useState<
+        Section[]
+    >([]);
+    const [ableToArchiveSections, setAbleToArchiveSections] = useState<
+        Section[]
+    >([]);
 
-    const [showAllSections, setShowAllSections] = useState(false);
     const { data: programResp, error: programError } = useSWR<
         ServerResponseOne<Program>,
         AxiosError
     >(`/api/programs/${id}`);
     const program = programResp?.data;
 
+    const {
+        data: sectionsResp,
+        error: sectionsError,
+        mutate: mutateSections
+    } = useSWR<ServerResponseMany<Section>, AxiosError>(
+        `/api/programs/${id}/sections`
+    );
+
+    const sections = sectionsResp?.data;
+
+    const checkResponse = useCheckResponse({
+        mutate: mutateSections,
+        refModal: archiveSectionsRef
+    });
+
+    useEffect(() => {
+        if (sections === undefined) return;
+        const unableToArchive = sections.filter(
+            (section) =>
+                selectedSections.includes(section.id) &&
+                (section.status === SelectedSectionStatus.Active ||
+                    section.status === SelectedSectionStatus.Scheduled) &&
+                section.enrolled_residents > 1
```

Comment:
Should this be >= 1? You're looking to make sure there is at least one resident enrolled, right?